fuzzinator/fuzzinator.ini

```diff
@@ -13,3 +13,4 @@
 db_uri=${fuzzinator.custom:db_uri}
 work_dir=${fuzzinator.custom:work_dir}
 cost_budget=${fuzzinator.custom:cost_budget}
+validate_after_update=${fuzzinator.custom:validate_after_update}
```

Comment:
Fuzzinator is not a SUT, it doesn't have this flag. Should be removed.

Reply:
Sorry, I was careless. :$